### ISSUE TITLE
GitLab build fails due to can't find node-sass

### ISSUE THREAD

**Issue by clevengr** [pc2ccs/pc2v9#302]
**Describe the issue**: 
  Merging code into a branch which forces a GitLab build (such as _**develop**_)  is producing GitLab build pipeline failures.

**To Reproduce**: 
  Merge any change into develop, then login to the PC2CCS project on GitLab (https://gitlab.com/pc2ccs/pc2v9)  (note: GitLAB, not GitHub) and select "CI/CD>Pipelines".  Wait for the latest pipeline to finish; it will show "Failed".
  Click on the "Failed" notification button to go to the pipeline, then click on the "job-build failed" button to open the "job-build" console.  Scroll down and find the place where the Angular build for WTI starts (command _**ngbuild:**_, around line 1220).  Scroll just below that to see the following:
```
    [exec] ERROR in ./src/styles/styles.scss (./node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/raw-css-loader.js!./node_modules/postcss-loader/src??embedded!./node_modules/sass-loader/lib/loader.js??ref--14-3!./src/styles/styles.scss)
     [exec] Module build failed (from ./node_modules/sass-loader/lib/loader.js):
     [exec] Error: Cannot find module 'node-sass'
```

**Expected behavior**: 
  Pushing a change to develop should result in a successful build on GitLab.  

**Actual behavior**: 
  Pushing a change to develop results in GitLab pipeline "job-build" step failing.

**Additional context**: 
  The most recent successful build was on September 5th; since then there have been no pushes to develop until this week, and those have failed.  This may have something to do with the fact that the build script (in particular, the package.json used by Angular) forces an update to the Node system.  Perhaps the new Node version (apparently,  v16?) is incompatible with the version of node-sass being loaded?  (In particular, https://github.com/sass/node-sass says that Node v16 requires Node-Sass V6.0+.   Need to check what versions of these two we're actually loading during the build process.

**Comment by lane55** [comment]
This issue has been resolved as of Feb 12.

Actions:  
1. reverted core-js to version ^2.5.4 in projects/WTI-UI/package.json (reverts #303)
2. Downgraded the NPM is a package manager (Node.js) version, from stable to 14.

**Comment by lane55** [comment]
The root cause was the latest stable release (v16.13.2) of node.js removed node-sass.
The previous successful build used version 14.17.6.
To restore node-sass, we use the latest version 14 build, aka v14.19.0.
   File: .gitlab-ci.yml, line 38 is now:   - n 14

**Comment by lane55** [comment]
Fixed in this build:   https://gitlab.com/pc2ccs/pc2v9/-/jobs/2081300104

**Comment by clevengr** [comment]
I downloaded nightly build PC² version 9.8build-6213.develop, released on 11 February 2022, and ran a smoke test.
I was able to bring up a contest, start WTI, login team1 and submit runs, start an AJ and judge them, and send a clar from Team1 and get back an answer.  I was also able to perform two simultaneous logins, from different machines, for Team1 and was able to submit runs from one login and see them from another.  Smoke test successful; closing this issue.
